Samples/WinMLSamplesGallery/WinMLSamplesGallery/Controls/DeviceComboBox.xaml.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.AI.MachineLearning;
+
+namespace WinMLSamplesGallery.Controls
+{
+    public sealed partial class DeviceComboBox : UserControl
+    {
+        public int SelectedIndex = 0;
+        public DeviceComboBox()
+        {
+            this.InitializeComponent();
+        }
+
+        private void changeSelectedIndex(object sender, RoutedEventArgs e)
+        {
+            SelectedIndex = DeviceBox.SelectedIndex;
+        }
+
+        public LearningModelDeviceKind GetDeviceKind()
+        {
+            if (SelectedIndex == 0)
+                return LearningModelDeviceKind.Cpu;
+            else
+                return LearningModelDeviceKind.DirectXHighPerformance;
```

Comment:
Just curious, but why DirectXHighPerformance over DirectX?

Reply:
No particular reason. Just following the precedent of @smk2007 Image Classifier sample but I think we have plans to add more options in the future